_posts/2020-04-27-release-2.4.0.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Release v2.4.0
+tags: ["v2.4.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 27/04 2020 the new version 2.4.0 is out
```

Comment:
This date should correspond with the release date.
And file name with this post publish date

Reply:
I should still get it there in time (per UTC ;-) )